include/base.php

```diff
@@ -163,7 +162,7 @@ public function switch_blog( $new_blog_id, $prev_blog_id ) {
 	 * @return bool
 	 */
 	protected function is_active_on_current_site(): bool {
-		return pll_is_plugin_active( POLYLANG_BASENAME ) && get_option( 'polylang' );
+		return pll_is_plugin_active( POLYLANG_BASENAME ) && ! empty( $this->options['version'] );
```

Comment:
Removed in [49d742f](https://github.com/polylang/polylang/pull/1481/commits/49d742faf6fe20d1ac2c47912e0e93078db7236c).
However, this could bring an edge case: doing `switch_to_blog()` after PLL being deactivated.

Reply:
Reverted in [2eb7ccf](https://github.com/polylang/polylang/pull/1481/commits/2eb7ccf53062f21e4ebff9bd9009aa3323e74d31).